GeneralsMD/Code/Libraries/Source/WWVegas/WW3D2/prim_anim.h

```diff
@@ -354,25 +354,25 @@ LERPAnimationChannelClass<T>::Evaluate (float time)
 	//
 	//	Don't interpolate past the last keyframe
 	//
-	if (time < m_Data[key_count - 1].Get_Time ()) {
+	if (time < this->m_Data[key_count - 1].Get_Time ()) {
```

Comment:
What are the errors relating to this? PrimitiveAnimationChannelClass<T> is a base class and m_Data and m_LastIndex are members of it and shouldn't require this-> to be resolved correctly.

Reply:
Use of this may not actually be needed anymore, see: https://github.com/TheSuperHackers/GeneralsGameCode/pull/547#discussion_r2027665787